### PR TITLE
[SPARK-8909][Documentation] Nice to have all the examples in scala, java,python, R to be same in sql-programming-guide

### PR DESCRIPTION
**Pull request by aloknsingh**
…ng-guide#Manually Specifying Options to be in sync with java,python, R version

### CODE REVIEW

File: docs/sql-programming-guide.md [hunk 1]
```diff
@@ -828,7 +828,7 @@ using this syntax.
 
 {% highlight scala %}
 val df = sqlContext.read.format("json").load("examples/src/main/resources/people.json")
-df.select("name", "age").write.format("json").save("namesAndAges.json")
+df.select("name", "age").write.format("parquet").save("namesAndAges.json")
```
Comment (andrewor14) [inline docs/sql-programming-guide.md]: the extension still says `.json`. Do we need to change that too?

Comment (aloknsingh) [inline docs/sql-programming-guide.md]: agree @andrewor14 . thanks for this :)

**Comment by sarutak** [comment]
ok to test.

**Comment by sarutak** [comment]
@aloknsingh Could you set more proper title like the corresponding issue in JIRA?

**Comment by aloknsingh** [comment]
@sarutak . Thanks for the comments. I have changed it to "Nice to have all the examples in scala, java,python, R to be same in sql-programming-guide" . Please feel free to change the title as you see right.

**Comment by sarutak** [comment]
@aloknsingh Sorry I mean we should have more proper title of this PR.

**Comment by rxin** [comment]
Thanks - I've merged this.

**Comment by aloknsingh** [comment]
@sarutak I have changed it as per your comments

**Comment by sarutak** [comment]
@aloknsingh Thanks :)

**Pull request closed without merging**